README.markdown

```diff
@@ -189,7 +189,7 @@ github "PromiseKit/MapKit" ~> 1.0
 
 ## Choose Your Networking Library
 
-`NSURLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
+`URLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
```

Comment:
Fair do’s. Like, I'm not sold on either way, so if you think it should be `URLSession` I'm cool with it.